### ISSUE TITLE
LAB derivations unclear, colors missing from "v5" list, "colors.csv" values are incorrectly labeled

### ISSUE THREAD

**Issue by phennessey** [aj90909/unofficial-pantone-solid-coated-2024-v5#1]
I see a number of issues with your repository and methodology which you may wish to address:

1. LAB is a larger color space than RGB, thus any values you're deriving from an RGB image of a color swatch from Pantone's site is NOT going to be accurate. You can convert from LAB to RGB, but you cannot actually go backwards. You should include a very clear explanation that the ACB file values are derived from the RGB values in the color swatch tiles you downloaded, and NOT the LAB values that Pantone created.

2. Your "V5" acb file appears to be missing some colors, as well as — quite confusingly — combining colors from the Pastel, Neon, and Metallic coated color groups. It's not a good idea to mash those all together in a single ACB, since without those additional labels, they are harder to understand. I recommend parsing those other color groups out, or at the very least labeling the color names appropriately in the output.

3. You've labeled the CSV output as "red green blue" — however this is not correct. They should be "L a b."

**Comment by Rickdraw** [comment]
Absolutely. This list is completely unreliable. I only proved it with a couple of formulas. 2350 C and 307 C. 

2350 C in this .acb is #952520 meanwhile on Pantone Connect the code is #AF231C. Also for 307 C, when it shows #3f6ca4 and the official is #006BA6. Plus, colors don't match at all w official ones.
